bin/dockerUtils.js

```diff
@@ -0,0 +1,252 @@
+// Inquirer used to Create Prompts in Terminal.
+import inquirer from "inquirer";
+
+export async function promptUserForServices() {
+  const services = [];
+
+  // Aggregating the answers for app service
+  const appAnswers = await inquirer.prompt([
+    {
+      type: "input",
+      name: "serviceName",
+      message: "Enter the name of the app service (default: app):",
+      default: "app",
+    },
```

Comment:
Take this from `package.json`. Question not needed.
We're already giving an option to customize name with `-n`.